client/src/customHooks/useAuth.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import jwtDecode from "jwt-decode";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+
+const useAuth = () => {
+  const navigate = useNavigate();
+  const [user, setUser] = useState(null);
+  const [clientId, setClientId] = useState("");
```

Comment:
`useState(null)` would be consistent unless we have a good reason. Empty strings in js are evil

Reply:
Fixed.